src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts

```diff
@@ -435,17 +435,22 @@ export default class ButtonView implements ButtonViewI {
         BUTTON_COLOR_CLASSES[this._buttonColor].HOVER_CLASS,
       );
       this._element.classList.remove('inboxsdk__button_hover');
+      this._element.classList.add(
```

Comment:
This fixes an issue I'm not sure why is happening. If we know we can fix it more properly.